pytorch_lightning/metrics/classification/auroc.py

```diff
@@ -47,10 +48,10 @@ class AUROC(Metric):
             this argument should not be set as we iteratively change it in the
             range [0,num_classes-1]
         average:
-            - ``'macro'`` computes metric for each class and uniformly averages them
-            - ``'weighted'`` computes metric for each class and does a weighted-average,
+            - ``AverageMethod.MACRO`` computes metric for each class and uniformly averages them
```

Comment:
no, we shall still preserve input as a string, `AverageMethod` is just internal tooling